### PR TITLE
create task for jobcenter-import

### PR DESCRIPTION
**Pull request by fabiangotzen**

### CODE REVIEW

File: playbooks/deploy-tasks.yml [hunk 1]
```diff
@@ -104,3 +104,63 @@
...
+        body: "name=avg-import-task&definition=avg-import"
+        headers:
+          Content-Type: "application/x-www-form-urlencoded"
+
+    - name: Register cron job avg import
+      cron:
+        name: "exec scdf avg-import-task"
+        minute: "0"
+        hour: "4"
+        job: "curl 'http://localhost:{{ spring.dataflow.http_port }}/tasks/executions' -i -X POST -d 'name=avg-import-task' -d arguments=start-date=`date +\\%FT\\%T`"
```
Comment (birom) [inline playbooks/deploy-tasks.yml]: Does the avg import use the start-date argument?

File: playbooks/deploy-tasks.yml [hunk 2]
```diff
@@ -104,3 +104,63 @@
...
+        body: "name=jobcenter-import-task&definition=jobcenter-import"
+        headers:
+          Content-Type: "application/x-www-form-urlencoded"
+
+    - name: Register cron job jobcenter import
+      cron:
+        name: "exec scdf jobcenter-import-task"
+        minute: "0"
+        hour: "4"
+        job: "curl 'http://localhost:{{ spring.dataflow.http_port }}/tasks/executions' -i -X POST -d 'name=jobcenter-import-task' -d arguments=start-date=`date +\\%FT\\%T`"
```
Comment (birom) [inline playbooks/deploy-tasks.yml]: Does the job-center import use the start-date argument?

File: playbooks/deploy-tasks.yml (hunk 1 above)
Comment (fabiangotzen) [reply playbooks/deploy-tasks.yml]: removed, thanks

File: playbooks/deploy-tasks.yml (hunk 2 above)
Comment (fabiangotzen) [reply playbooks/deploy-tasks.yml]: removed, thanks

**Comment by fabiangotzen** [comment]
Makes sense. I've created a Technical Debt Task (DF-905) in our backlog to extract this.

**Pull request merged**